web/viewer.css

```diff
@@ -1703,6 +1703,22 @@ html[dir='rtl'] #documentPropertiesOverlay .row > * {
   .secondaryToolbarButton.documentProperties::before {
     content: url(images/secondaryToolbarButton-documentProperties@2x.png);
   }
+
+  .outlineItemToggler::before {
+    -webkit-transform: scale(0.5);
+    transform: scale(0.5);
+    top: -1px;
+    content: url(images/treeitem-collapsed@2x.png);
+  }
+  .outlineItemToggler.outlineItemsHidden::before {
+    content: url(images/treeitem-expanded@2x.png);
+  }
+  html[dir="ltr"] .outlineItemToggler::before {
+    right: 0px;
```

Comment:
Zero does not need a unit, so remove it. This would also be consistent with most of the code base.

```
web$ grep ' 0;' viewer.css  | wc -l
81
web$ grep ' 0px;' viewer.css  | wc -l
8
```